README.md

```diff
@@ -47,40 +43,48 @@ and / or
 
 - Add some global options when running the daemon (run `teosd.py -h` for more info).
 
-## Running TEOS in another network
-
-By default, `teos` runs on `mainnet`. In order to run it on another network you need to change the network parameter in the configuration file or pass the network parameter as a command line option. Notice that if teos does not find a `bitcoind` node running in the same network that it is set to run, it will refuse to run.
+### Tower identity and signing key
 
+In order to run `teos`, you need to have a pair of keys stored in your data directory, that are used as the tower's identity. Users can recognize the tower by its public key, and the tower uses the private key to sign appointments. You can follow [generate keys](#generate-keys) to generate them.
```

Comment:
I'd change it for:

`teos` needs a pair of keys that will serve as tower id and signing key. The former can be used by users to identify the tower, whereas the latter is used by the tower to sign responses. You can follow [generate keys](#generate-keys) to generate them.

Reply:
Done in ada9645.